Open Judge System/Web/OJS.Web/Areas/Contests/ViewModels/Results/ProblemResultPairViewModel.cs

```diff
@@ -2,14 +2,12 @@
 {
     public class ProblemResultPairViewModel
     {
-        public int Id { get; set; }
+        public int ProblemId { get; set; }
 
         public bool ShowResult { get; set; }
 
-        public BestSubmissionViewModel BestSubmission { get; set; }
-
-        public string ProblemName { get; set; }
+        public int MaximumPoints { get; set; }
 
-        public int ProblemOrderBy { get; set; }
```

Comment:
Why did we remove the OrderBy? Is it not used?

Reply:
It was used only for ordering the problems in the view model, but now I order the problems before mapping them to the view model